apps/api/src/domain/models/song/commands/create-song.command.ts

```diff
@@ -34,4 +35,7 @@ export class CreateSong implements ICommand {
 
     @NonNegativeFiniteNumber()
     readonly lengthMilliseconds: number;
+
+    @RawData()
+    readonly rawData?: Record<string, unknown>;
```

Comment:
unknown has safer type safety than any, correct?

Reply:
Exactly. You can only "dot into" properties on `unknown` after using a type guard to assert that the property should actually exist. With `any` you can do `any.bogusProp` and the compiler won't complain.